scripts/globals/spells/black/aspir.lua

```diff
@@ -37,7 +37,7 @@ spellObject.onSpellCast = function(caster, target, spell)
 
     dmg = dmg * xi.settings.main.DARK_POWER
 
-    if target:isUndead() or target:hasImmunity(xi.immunity.ASPIR) then
+    if target:isUndead() or target:hasImmunity(8192) then
```

Comment:
Can you correct the ENUM being used here instead?

Reply:
Arguments in this function only take INT values. It doesn't seem like the xi.immunity is being used anywhere else. We could instead have these return decimal instead of hex